components/notebook-controller/controllers/notebook_controller.go

```diff
@@ -215,6 +216,31 @@ func (r *NotebookReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 		return ctrl.Result{}, err
 	}
 
+	// Check if annotations for the Notebook instance imply a requirement for notebook restart.
+	// This annotation is supposed to be added once a ConfigMap used in the notebook is added/updated.
+
+	annotations := instance.GetAnnotations()
```

Comment:
```suggestion
	annotations := instance.ObjectMeta.GetAnnotations()
```
https://pkg.go.dev/k8s.io/apimachinery@v0.17.0/pkg/apis/meta/v1#ObjectMeta.GetAnnotations

Reply:
We should be able to use both `instance.GetAnnotations()` and `instance.ObjectMeta.GetAnnotations()` as `ObjectMeta` is embedded in `Notebook` struct. Wdyt?